docs/DafnyRef/UserGuide.md

```diff
@@ -365,7 +365,18 @@ The command emits exit codes of
 
 #### 25.5.1.9. `dafny format` {#sec-dafny-format}
 
-This command is not yet released, but will be a command that formats source code to a consistent style.
+Dafny supports a formatter, which for now only changes the indentation of every line in a Dafny file, so that it conforms
+to the idiomatic Dafny code formatting style.
+For the formatter to work, the file should be parsed correctly by Dafny.
+
+There are four ways to use the formatter:
+
+* `dafny format file1.dfy file2.dfy ...` formats each file provided in arguments.
+* `dafny format --print:- file1.dfy file2.dfy ...` format each file but instead of writing files, output their formatted content in stdout
+* `dafny format` Look up for all the Dafny files recursively in the current repository and format them.
```

Comment:
No -- being in a repo is a special case. I suggest the arguments be files or directories, and use something 
like --repo if one really needs to format the repo (without having to write the directory for it)

Reply:
What do you mean by "needs to format the repo without having to write the directory for it"? One could always explicitly pass "." which is shorter than "--repo".

Reply:
That's a correct guess ! Thanks for clarifying your point.